secator/celery.py

```
@@ -144,7 +144,7 @@ def chunker(seq, size):
 
 
 @app.task(bind=True)
-def start_runner(self, config, targets, results=[], run_opts={}, hooks={}, validators={}, context={}):
+def start_runner(self, config, targets, results=[], run_opts={}, hooks={}, drivers=[], validators={}, context={}):
```

Ruff (0.15.15)

[warning] Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)